ksql-engine/src/test/java/io/confluent/ksql/integration/StreamsSelectAndProjectIntTest.java

```diff
@@ -326,7 +344,7 @@ private void testSelectWithFilter(String resultStream,
 
   private void createOrdersStream() throws Exception {
     ksqlContext.sql(String.format("CREATE STREAM %s (ORDERTIME bigint, ORDERID varchar, ITEMID "
-                                  + "varchar, ORDERUNITS double, PRICEARRAY array<double>,"
+                                  + "varchar, ORDERUNITS double, TIMESTAMP varchar, PRICEARRAY array<double>,"
                                   + " KEYVALUEMAP "
                                   + "map<varchar, double>) WITH (kafka_topic='%s', "
                                   + "value_format='JSON', key='ordertime');",
```

Comment:
Maybe add a test to set the source timestamp in `CREATE STREAM` or `CREATE TABLE` DDL statements.

Reply:
yep - thought we already had that for some reason